solidity/contracts/bridge/Bridge.sol

```diff
@@ -1227,6 +1284,62 @@ contract Bridge is Ownable {
         bank.transferBalance(treasury, outputsInfo.totalTreasuryFee);
     }
 
+    /// @notice Submits a wallet fraud challenge.
+    /// @param walletPublicKey The public key of the wallet in the uncompressed
+    ///                        and unprefixed format (64 bytes).
+    /// @param walletPubKeyHash 20-byte public key hash (computed using Bitcoin
+    ///        HASH160 over the compressed ECDSA public key) of the wallet which
+    ///        performed the redemption transaction.
+    /// @param sighash Hash of the data (preimage) used for generating signature
+    ///                for the given input.
+    /// @param v Signature recovery value.
+    /// @param r Signature r value.
+    /// @param s Signature s value.
+    function submitFraudChallenge(
```

Comment:
Worth adding a `@dev` section with requirements listed. See `@dev` for `requestRedemption` and friends.

Reply:
Done in e1099ea68609ac8c6d083d763014e62653af65c6